packages/keycodes/src/index.js

```diff
@@ -144,6 +144,8 @@ export const SHIFT = 'shift';
  */
 export const ZERO = 48;
 
+export { isAppleOS };
```

Comment:
Not completely convinced about exporting this from `keycodes`. Maybe there should be a `@wordpress/platform` package or something.

It's a very minor thing though, so I'll leave it up to you to decide what to do 👍

Reply:
Yes, I was wondering also if it's a good idea. But I looked through all the apple checks we're making in Gutenberg, and they're always related to the keyboard, so it seems to make sense as part of this package.